AUTHORS.txt

```diff
@@ -0,0 +1 @@
+Omar Awile (maintainer)
```

Comment:
Now in theory, if you contribute to this, do you have to list yourself?

Reply:
True 😂